seatunnel-e2e/seatunnel-connector-v2-e2e/connector-cdc-mysql-e2e/src/test/resources/mysqlcdc_to_mysql_with_multi_table_mode_one_table.conf

```diff
@@ -20,19 +20,22 @@
 
 env {
   # You can set engine configuration here
-  parallelism = 1
+  parallelism = 3
   job.mode = "STREAMING"
   checkpoint.interval = 5000
 }
 
 source {
   MySQL-CDC {
     result_table_name = "customers_mysql_cdc"
-    server-id = 5652
-    username = "mysqluser"
+    server-id = 5652-5660
+    username = "st_user_source"
     password = "mysqlpw"
     table-names = ["mysql_cdc.mysql_cdc_e2e_source_table"]
     base-url = "jdbc:mysql://mysql_cdc_e2e:3306/mysql_cdc"
+
+    snapshot.split.size = 1
+    snapshot.fetch.size = 1
```

Comment:
init parallelism=3 and snapshot.split.size = 1

Make sure that all 3 readers can open jdbc connections to read data